dev/release/README.md

```diff
@@ -229,6 +210,10 @@ The vote will be open for at least 72 hours.
 [ ] +0
 [ ] -1 Do not release this as Apache Arrow DataFusion 5.1.0 because...
 
+Here is my vote:
+
++1
```

Comment:
Do you want to add this to the scripts as well? Or maybe you already did

Reply:
Good point. I hadn't thought of that. Will update shortly ...